src/JSInterop/Microsoft.JSInterop/src/JSRuntimeBase.cs

```diff
@@ -80,6 +81,10 @@ internal void EndInvokeDotNet(string callId, bool success, object resultOrExcept
             {
                 resultOrException = resultOrException.ToString();
             }
+            else if (!success && resultOrException is ExceptionDispatchInfo edi)
```

Comment:
nit: Isn't `resultOrException is ExceptionDispatchInfo edi` enough test?

Reply:
This is similar to the test above for exceptions. I don't see a reason to change this since there aren't many tests for exceptions in general.